database/articles.js

```diff
@@ -1,42 +1,55 @@
 'use strict';
 
 const express = require('express');
-const expressApp = express();
-const bodyParser = require('body-parser');
-const jsonParser = bodyParser.json();
+const expressApp = express.Router();
 
-let articles = [];
+let articles = { articles: [] };
 
 const getAllArticles = () => {
   return articles;
 };
 
+const getSpecificArticle = (req) => {
```

Comment:
for the getSpecificArticle function the req parameter has the value of req.url since I pass in req.url as the parameter going to the getSpecificArticle function in my routes/article.js file.

Attempting to change the parameter for the getSpecificArticle function to req.url turns the name white breaking my function. Should I declare req.url as a global variable in my routes/article.js file and pass that instead?

Reply:
No. What I'm saying is that if you name your function parameter `req`. I'm expecting that to be the entire request object. If you're just passing in the url, then you should rename the function parameter to something more like `reqUrl` or `titleUrl`

Reply:
got it